lib/plugins.js

```diff
@@ -232,7 +233,7 @@ exports.init = (config, onCompile) => {
   speed.profile('Loaded plugins');
   const includes = getPluginIncludes(plugins);
   return Promise.resolve({
-    compilers, linters, includes, teardownBrunch, optimizers,
+    compilers, staticCompilers, linters, includes, teardownBrunch, optimizers,
```

Comment:
static compilers is a new category of compilers, ones which implement `compileStatic`